src/server/router/page.js

```diff
@@ -0,0 +1,110 @@
+import config from "../config";
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import {ServerStyleSheet} from "styled-components";
+import headerProps from "./componentProps/header";
+
+export default class Page {
+	constructor(app, details, RootComponent) {
+		this.RootComponent = RootComponent;
+
+		this.details = details;
+		this.props = {}; // Should be filled by the page's _getLatestProps function
+
+		// Set up route in express
+		app.get(`/${details.id}`, (req, res) => {
+			this._renderPage(req, res);
+		});
+
+		// Add route to header props
+		headerProps.route = details.id;
```

Comment:
This modifies the `headerProps` module, meaning all pages are affected by the page that changes this last.

Reply:
oh oopsie poopsie